core/src/main/java/org/apache/calcite/rex/RexUtil.java

```diff
@@ -727,6 +727,13 @@ private static RexNode deref(@Nullable RexProgram program, RexNode node) {
       return true;
     }
 
+    @Override public Boolean visitNamedParam(RexNamedParam namedParam) {
+      // TODO: Confirm this is correct
```

Comment:
This makes sense, I think it is correct.

Reply:
Agreed.